Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocListNavigationBlockRenderer.cs

```diff
@@ -207,7 +207,7 @@ private void RenderPageInformation (BocListRenderingContext renderingContext)
       renderingContext.Writer.RenderEndTag(); // end span
 
       string script = string.Format (
-          "BocList.InitializeNavigationBlock ($('#{0}'), $('#{1}'))",
+          "BocList.InitializeNavigationBlock (document.getElementById('{0}'), document.getElementById('{1}'))",
```

Comment:
`BocList.InitializeNavigationBlock` should use the HTMLElement or CSS selector pattern so that we can use a css selectors here. See handbook, under concept "jQuery objects as parameters"